src/main/java/success/planfit/domain/RefreshToken.java

```diff
@@ -0,0 +1,29 @@
+package success.planfit.domain;
+
+
+import jakarta.persistence.*;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import success.planfit.domain.user.User;
+
+@Getter
+@NoArgsConstructor
+@Entity
+public class RefreshToken {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @JoinColumn (nullable = false)
+    @ManyToOne(fetch = FetchType.LAZY)
+    private User user;
```

Comment:
```suggestion
    @JoinColumn(nullable = false)
    @OneToOne(fetch = FetchType.LAZY)
    private User user;
```

리프레쉬 토큰과 회원은 1:1 관계라서 `@OneToOne`으로 수정해 주시면 감사하겠습니다!